commercetools-models/src/main/java/io/sphere/sdk/customers/commands/updateactions/ChangeAddress.java

```diff
@@ -18,22 +18,28 @@
 public final class ChangeAddress extends UpdateActionImpl<Customer> {
     private final Address address;
     private final String addressId;
+    private final String addressKey;
 
-    private ChangeAddress(final String addressId, final Address address) {
+    private ChangeAddress(final String addressId, final Address address, String addressKey) {
         super("changeAddress");
         this.address = address;
         this.addressId = addressId;
+        this.addressKey = addressKey;
     }
 
     public static ChangeAddress of(final String addressId, final Address address) {
-        return new ChangeAddress(addressId, address);
+        return new ChangeAddress(addressId, address, address.getKey());
```

Comment:
This will not work out.

It must be either addressId **or** addressKey

See https://docs.commercetools.com/api/projects/customers#address-selection

So if the Id is given you have to set addressKey to null and vice versa.